packages/arb-token-bridge-ui/src/components/TransactionHistory/TransactionsTable/TransactionsTable.tsx

```diff
@@ -82,14 +160,17 @@ export function TransactionsTable({
   pageParams,
   setPageParams,
   transactions,
+  isSmartContractWallet,
```

Comment:
TransactionsTable had props related to its own properties. I am thinking should we have `isSmartContractWallet` extracted within the component using `const { isSmartContractWallet } = useAccountType()` and not pass it as props. 
Otherwise, it opens the question of why not have all state variables passed down as props.

Reply:
I had it done like this originally, but the problem is with `useAccountType()` being undefined between renders. If the tx history table re-renders then we have UI side effects like rounded edges disappearing, components flashing etc.

When passing it as prop we have persistent value. I agree, I'd prefer not to pass it but dealing with UI with account type isn't the best.